source/Sagan-RDBMS-Tests/SingleSessionRDMBSRepositoryTest.class.st

```diff
@@ -201,12 +201,15 @@ SingleSessionRDMBSRepositoryTest >> testLogging [
 	loggingAsserter := LoggingAsserter on: self.
 	loggingAsserter
 		runMemoryLoggerDuring: [ self extraterrestrials store: self silvesterStallone ];
-		assertLogRecordsMatch: #( 
+		assertLogRecordsMatchInTheFollowingOrder: #( 
 		'[DEBUG] Glorp event received {"detail":"Login"}'
 		'[DEBUG] Glorp event received {"detail":"Login finished"}'
+		'[DEBUG] Glorp event received {"detail":"Begin Transaction"}'
 		'[DEBUG] Glorp event received {"detail":"INSERT INTO Extraterrestrial (sequential_number,first_name,last_name,birth_date)  VALUES (1,''Silvester'',''Stallone'',''1946-07-06'')"}'
 		'[DEBUG] Glorp event received {"detail":"Commit Transaction"}'
 		) .
+		
+	loggingAsserter stopLoggers.
```

Comment:
Use an `ensure:` here so if the previous steps fail, it wouldn't let dangling logger instances around. Or you can also create a method like `withLoggingAsserterDo:` that wraps the asserter creation and ensures to stop loggers at the end.